### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by ZeroerWiser** [DavidZWZ/LightGODE#1]
log shows that the training loss of the model in the epochs are all negative

**Comment by DavidZWZ** [comment]
Hi,

Thank you for your interest in LightGODE! Please note that during training, we use the [DirectAU](https://dl.acm.org/doi/abs/10.1145/3534678.3539253) [1] loss function for optimization. It's important to note that the uniformity component of the loss may become negative, especially when users and items in the batch are more distantly related. However, as long as the overall loss continues to decrease during training, this behavior is expected and should not affect the effectiveness of LightGODE.

[1] Wang C, Yu Y, Ma W, et al. Towards representation alignment and uniformity in collaborative filtering[C]//Proceedings of the 28th ACM SIGKDD conference on knowledge discovery and data mining. 2022: 1816-1825.
